File: src/main/java/Version_Java/Juice.java
package Version_Java;

import java.time.LocalDate;

public class Juice extends Drinks{
    public Juice(String name, double cost, LocalDate date) {
        super(name, cost, date, 2);
    }

    @Override
    public String toString() {
        return "Version_Java.Beer{" +
                "name='" + name + '\'' +
                ", cost=" + cost +
                ", date=" + date +
                ", expiration=" + expiration +
                '}';
    }

    @Override
    public boolean equals(Object obj) {
        return super.equals(obj);
    }
}

File: src/main/java/Version_Kotlin/Juice.kt
package Version_Kotlin

import java.time.LocalDate

class Juice (name: String, cost: Double, date: LocalDate) : Drinks(name, cost, date, 2){
    override fun toString() ="Version_Java.Beer{" +
                "name='" + name + '\'' +
                ", cost=" + cost +
                ", date=" + date +
                ", expiration=" + expiration +
                '}'

    override fun hashCode(): Int {
        var result = name.hashCode()
        result = 31 * result + cost.hashCode()
        result = 31 * result + date.hashCode()
        result = 31 * result + expiration
        return result
    }

    override fun equals(o: Any?): Boolean {
        return super.equals(o)
    }
}